WeScan/Scan/RectangleFeaturesFunnel.swift

```diff
@@ -90,7 +91,7 @@ final class RectangleFeaturesFunnel {
             bestRectangle.rectangleFeature.isWithin(matchingThreshold, ofRectangleFeature: previousRectangle) {
         } else if bestRectangle.matchingScore >= minNumberOfMatches {
             completion(bestRectangle.rectangleFeature)
-        }
+        }*/
```

Comment:
If we don't need this code anymore, can we get rid of it?

Reply:
Which code?

Reply:
AvdLee is referring to the dead code (code commented out).

Reply:
Ah, I understand. Not using it seems to work on iOS 10, but again, I haven't had a chance to test on iOS 11. As soon as I do, i'll either leave it in/take it out.

Reply:
Update on this: the commented out code makes the rectangles very jerky on all versions of iOS. I'm looking at reimplementing this code soon.

Reply:
Re-added with some changes. See commit f1e01c1.